lib/IRGen/TBDGen.cpp

```diff
@@ -244,6 +244,8 @@ getLinkerPlatformId(OriginallyDefinedInAttr::ActiveVersion Ver,
     llvm_unreachable("cannot find platform kind");
   case swift::PlatformKind::OpenBSD:
     llvm_unreachable("not used for this platform");
+  case swift::PlatformKind::FreeBSD:
+    llvm_unreachable("not used for this platform");
```

Comment:
Can we coalesce? (Also, nitpick, the order of FreeBSD relative to OpenBSD is inconsistent. It's not important in any practical sense, but consistency is a good thing.)